src/docs/transactions.dox

```diff
@@ -54,6 +54,12 @@ discarding any cursor position as well as any key and value.
 
 @snippet ex_all.c transaction commit/rollback
 
+Applications can call WT_SESSION::reset_snapshot to reset snapshots for
+snapshot isolation transactions to update their existing snapshot. It raises
```

Comment:
I am thinking of adding a sentence of the use of this API. How about adding something like this `Reset snapshot will avoid pinning the content in the database that is no longer needed`.

Reply:
I second that, it will be useful to say what resetting snapshot will be used for and what are disadvantages if any of resetting snapshot.

Reply:
Updated the documentation